### DISCUSSION TITLE
삭제 관련 건

### DISCUSSION THREAD

**Discussion by woodimora** [Scope-Team9/Scope_BackEnd#84] [General]
멘토링 시간에 받았던 피드백으로 테이블을 삭제하는 대신 deletedDate 로 표현하는 것에 대한 의견을 듣고싶습니다.

**Comment by 15011106** [comment]
현업에서 사용하는 방식은 한번 적용해보는게 좋을 것 같습니다. 구현 후 deleteDate로 바꾸는건 어떨까요 !

**Comment by 9sanha** [comment]
RDS를 사용하지 않고 MYSQL만 사용한다면 좋을 것 같습니다.

**Reply by 15011106** [reply]
RDS -> 로컬 변경이유는 비용문제인가요 ?
